src/components/overrides/SiteTitle.astro

```diff
@@ -14,13 +14,9 @@ type Link = {
 
 const links: Link[] = [
 	{
-		title: 'Guides',
-		value: '/guides',
-	},
-	{
-		title: 'References',
-		value: '/references',
-	},
+		title: 'Docs',
+		value: '/docs',
+	}
```

Comment:
```suggestion
	},
```

Reply:
haha I never expected someone to fix my things while fixing it myself 😂

Reply:
lmao, sorry, I know it's a draft and couldn't hold myself haha, I'll refrain from looking any further 😂